sdks/unreal/Agones/Source/Agones/Private/AgonesComponent.cpp

```diff
@@ -146,6 +192,45 @@ void UAgonesComponent::GameServer(const FGameServerDelegate SuccessDelegate, con
 	Request->ProcessRequest();
 }
 
+bool UAgonesComponent::DeserializeAndBroadcastWatch(FString const& JsonString)

```

Comment:
```suggestion
void UAgonesComponent::DeserializeAndBroadcastWatch(FString const& JsonString)
```

`bool` doesnt seem to be used here